frontend/src/static/css/style.scss

```diff
@@ -266,6 +266,14 @@ header {
 
 /* Commit Message Body in Zoom Tab */
 .commit-message {
+  padding: 5px;
+
+  &:focus {
+    .tag{
+      border: mui-color('blue', '500') solid;
```

Comment:
can use the same colour as the border for the highlighted commit?

Reply:
done

Reply:
The colour still looks different tho. Is it possible to use the default blue colour for the tag?

<img width="543" alt="Screen Shot 2020-02-15 at 7 56 38 PM" src="https://user-images.githubusercontent.com/32864116/74587461-8ac2e400-502d-11ea-8c0d-25542535e124.png">

Reply:
Currently, clicking the tag within the commit message also focus on the commit:

![Kapture 2020-02-15 at 20 03 15](https://user-images.githubusercontent.com/32864116/74587506-4be15e00-502e-11ea-935b-2ecddaa213b7.gif)

Reply:
This is kinda the additional functionality. You can select a commit message (by clicking anywhere within the commit-message div) and it too gets highlighted.

Reply:
> The colour still looks different tho. Is it possible to use the default blue colour for the tag?

The border color all forced to #2196f3 alr, I feel should be the screen display problem.
![Screenshot (16)](https://user-images.githubusercontent.com/39231750/74587594-bc3caf00-502f-11ea-826d-324206913445.png)
![Screenshot (17)](https://user-images.githubusercontent.com/39231750/74587595-be9f0900-502f-11ea-91a4-aefb6fef0344.png)

Reply:
If it is truly a problem, can consider the bind style functionality of vue?

Reply:
Dont think it is truly a problem. On my computer the preview page the color look the same.
![image](https://user-images.githubusercontent.com/39231750/74647097-03ed4300-51b6-11ea-9230-a94cda52b4b0.png)